build-system/tasks/visual-diff/index.js

```diff
@@ -702,12 +702,14 @@ visualDiff.description = 'Run the AMP visual diff tests';
 visualDiff.flags = {
   'main': 'Include a blank snapshot (baseline for skipped builds)',
   'empty': 'Create a dummy Percy build with only a blank snapshot',
+  'esm': 'Serve ESM JS (uses the new typescript server transforms)',
```

Comment:
I have a feeling this `new` thing would become quite old by the time anyone bothers to remove this parenthesis. Is there a strong reason to keep this comment in the flag description?

Reply:
no reason. will remove